system/config/Config.cfc

```diff
@@ -310,8 +310,7 @@ component {
 
 		settings.filters = {
 			livePages = {
-				  filter       = "page.trashed = '0' and page.active = '1' and ( page.embargo_date is null or :now > page.embargo_date ) and ( page.expiry_date is null or :now < page.expiry_date )"
-				, filterParams = { "now" = { type="cf_sql_date", value=Now() } }
+				filter = "page.trashed = '0' and page.active = '1' and ( page.embargo_date is null or NOW() > page.embargo_date ) and ( page.expiry_date is null or NOW() < page.expiry_date )"
```

Comment:
The only issue with using raw Now() in the SQL is database engine compatibility. There is a database adapter method for getting the 'now' function for the given database engine being used, but that is not possible to access in a static filter.

I think the real fix here is the param type. I think it needs to be something like `cf_sql_timestamp` rather than `cf_sql_date`. Can you try that?